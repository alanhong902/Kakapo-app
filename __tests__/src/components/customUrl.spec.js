import React from 'react';
import { CustomUrl } from 'components/import/customUrl';
import { getData, createComponentWithIntl } from '../helper';

test('<CustomUrl/>', () => {
  const props = {
    ...getData('themes'),
  };
  const tree = createComponentWithIntl(<CustomUrl {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
