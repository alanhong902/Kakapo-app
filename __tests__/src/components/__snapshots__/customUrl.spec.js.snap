// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomUrl/> 1`] = `
<div
  className="customurl"
>
  <h5>
    <span>
      Add a custom stream
    </span>
  </h5>
  <form
    onSubmit={[Function]}
  >
    <div
      className="media-import"
    >
      <div
        className="group"
      >
        <input
          autoComplete="off"
          name="name"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          type="text"
        />
        <label
          htmlFor="name"
        >
          <span>
            Choose a name for this sound
          </span>
        </label>
        <span
          className="highlight"
        />
        <span
          className="bar"
        />
      </div>
      <div
        className="group"
      >
        <input
          autoComplete="off"
          name="url"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={true}
          type="text"
        />
        <label
          htmlFor="url"
        >
          <span>
            Enter a link to an audio file
          </span>
        </label>
        <span
          className="highlight"
        />
        <span
          className="bar"
        />
      </div>
      <button
        className="button"
        style={
          Object {
            "backgroundColor": "#4CAF50",
            "borderColor": "#4CAF50",
          }
        }
        type="button"
      >
        <span>
          Save
        </span>
      </button>
    </div>
  </form>
</div>
`;
